notify/mail.sh

```diff
@@ -10,6 +14,107 @@ mail_send() {
   _debug "_content" "$_content"
   _debug "_statusCode" "$_statusCode"
 
-  _err "Not implemented yet."
-  return 1
+  unset -f _MAIL_BIN _MAIL_BODY _MAIL_CMD
```

Comment:
you don't need `unset` anything,   the `mail_send()` function should be called in an isolated shell, so, the variable should be empty already.  please double check.